common/src/main/java/io/netty/util/internal/Cleaner0.java

```diff
@@ -29,10 +29,9 @@
  *
  * For more details see <a href="https://github.com/netty/netty/issues/2604">#2604</a>.
  */
-final class Cleaner0 {
+final class Cleaner0 implements Cleaner {
```

Comment:
It's a possible deadlock such as [guava issue #1977](https://github.com/google/guava/issues/1977)?